kms/encrypt_symmetric.go

```diff
@@ -40,21 +41,41 @@ func encryptSymmetric(w io.Writer, name string, message string) error {
 	// Convert the message into bytes. Cryptographic plaintexts and
 	// ciphertexts are always byte arrays.
 	plaintext := []byte(message)
+	
+	// Optional, but recommended: Compute plaintext's CRC32C.
+	// See crc32c() function below.
+	plaintextCrc32c = crc32c(plaintext)
 
 	// Build the request.
 	req := &kmspb.EncryptRequest{
 		Name:      name,
 		Plaintext: plaintext,
+		PlaintextCrc32C: plaintextCrc32c,
 	}
 
 	// Call the API.
 	result, err := client.Encrypt(ctx, req)
 	if err != nil {
 		return fmt.Errorf("failed to encrypt: %v", err)
 	}
+	
+	// Optional, but recommended: perform integrity verification on result.
+        // For more details on ensuring E2E in-transit integrity to and from Cloud KMS visit:
+        // https://cloud.google.com/kms/docs/data-integrity-guidelines
```

Comment:
Looks like you need to run `gofmt -w .`.

Reply:
Done.